docs/versioned_docs/version-3.5.0-LTS/setup/env-vars.md

```diff
@@ -48,6 +48,13 @@ Replace `username`, `password`, `hostname`, `port`, and `database_name` with you
 - `TOOLJET_DB_PASS`: Database password
 - `TOOLJET_DB_PORT`: Database port
 
+#### Why ToolJet Requires Two Databases
```

Comment:
Could we remove this table format. It seem odd since others do not have table. It would be better if we can explain it without puting in table format.